custom_components/waste_collection_schedule/waste_collection_schedule/source/olo_sk.py

```diff
@@ -0,0 +1,166 @@
+import datetime
+from waste_collection_schedule import Collection
+import requests
+
+TITLE = "OLO"  # Title will show up in README.md and info.md
+DESCRIPTION = "Source script for olo.sk"  # Describe your source
+# Insert url to service homepage. URL will show up in README.md and info.md
+URL = "https://www.olo.sk"
+TEST_CASES = {  # Insert arguments for test cases to be used by test_sources.py script
+    "Jantarova 47": {"street": 'Jantarova 47', "registrationNumber": '2441788'},
+    "Jasovska 8": {"street": 'Jasovska 8', "registrationNumber": '1353013'},
+    "Rovniankova 5": {"street": 'Rovniankova 5', "registrationNumber": ''},
+}
+
+API_URL = "https://olo-strapi-meilisearch.bratislava.sk/indexes/waste-collection-day/search"
+ICON_MAP = {   # Optional: Dict of waste types and suitable mdi icons
+    "DOMESTIC": "mdi:trash-can",
+    "RECYCLE": "mdi:recycle",
+    "KITCHEN": "mdi:coutertop",
+}
+
+WEKDAYS = {
+    "pondelok": "Monday",
+    "utorok": "Tuesday",
+    "streda": "Wednesday",
+    "štvrtok": "Thursday",
+    "piatok": "Friday",
+    "sobota": "Saturday",
+    "nedeľa": "Sunday"
+}
+
+#### Arguments affecting the configuration GUI ####
+
+HOW_TO_GET_ARGUMENTS_DESCRIPTION = {  # Optional dictionary to describe how to get the arguments, will be shown in the GUI configuration form above the input fields, does not need to be translated in all languages
+    "en": "HOW TO GET ARGUMENTS DESCRIPTION"
+}
+
+PARAM_DESCRIPTIONS = {  # Optional dict to describe the arguments, will be shown in the GUI configuration below the respective input field
+    "en": {
+        "street": "Street name",
+        "registrationNumber": "Registration number"
+    }
+}
+
+PARAM_TRANSLATIONS = {  # Optional dict to translate the arguments, will be shown in the GUI configuration form as placeholder text
+    "en": {
+        "street": "Enter street name",
+        "registrationNumber": "Enter registration number"
+    }
+}
+
+#### End of arguments affecting the configuration GUI ####
+
+
+class Source:
+    # argX correspond to the args dict in the source configuration
+    def __init__(self, street: str, registrationNumber: str):
+        self._street = street
+        self._registrationNumber = registrationNumber
+
+    def parseDates(self, dates_str: str) -> list[datetime.date]:
+        dates = dates_str.split(";")
+        dates = [date.strip() for date in dates]
+        dates = [date for date in dates if date]
+        return [datetime.datetime.strptime(date, "%d.%m.%Y").date() for date in dates]
+
+    def fetchWasteType(self, waste_type: str) -> str:
+        headers = {
+            "Content-Type": "application/json",
+            "Accept": "application/json",
+            "authorization": "Bearer ae84ae0982c2162a81eb253765ceaa8593abd9105c71954cf5c9620b0178cbb6"
```

Comment:
Is there another way to get this header, I don't like hard coding a Bearer token